scripts/configs/example.yaml

```diff
@@ -3,15 +3,15 @@
 APPROACHES:
   my-oracle:  # used in constructing the experiment ID
     NAME: "oracle"
-    FLAGS: []
+    FLAGS: {}
   nsrt-learning:
     NAME: "nsrt_learning"
     FLAGS:
       disable_harmlessness_check: True  # just an example
 ENVS:
   my-cover:  # used in constructing the experiment ID
     NAME: "cover"
-    FLAGS: []
+    FLAGS: {}
   single-block-cover:
```

Comment:
i was a bit confused by this naming and how it relates to "my-cover" above. it might be clearer if you change the names, like:
"my-cover" -> "cover-default-settings"
"single-block-cover" -> "cover-single-blocks"